org.libredirect.frontends_manager.metainfo.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.libredirect.frontends_manager</id>
+  
+  <name>Libredirect Frontends Manager</name>
+  <summary>Run privacy frontends on your computer</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  
+  <description>
+    <p>
+      Run privacy-friendly frontends locally on your computer rather than relying on 3rd party instances. A much better trust-less model. For anonymizing your IP, use Lokinet or a VPN.
```

Comment:
Nitpick: Please try to follow the quality guidelines eg.

https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines#no-weird-formatting-1

> summary [...] shouldn't end with a full stop

Not a technical requirement but improves presentation.

Reply:
Formatted it better.